docs/DeveloperGuide.md

```diff
@@ -325,7 +349,97 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use case (UC3) : Schedule an appointment for the patient**
+
+**MSS**
+
+1.  User requests to list persons
+2.  AddressBook shows a list of persons
+3.  User requests to schedule an appointment for a specific person in the list
+4.  AddressBook prompts User to input appointment details
+5.  User inputs appointment details 
+6.  AddressBook schedules appointment for the patient
```

Comment:
Makes sense! Just wondering, what if there is a new patient? Would we first have to register the person in our list (in UC1), then schedule an appt for them?

Reply:
That's true, will add that as a precondition 👍